source/oauth/settings.d

```diff
@@ -194,9 +194,9 @@ class OAuthSettings
         foreach (k, v; extraParams)
             reqParams[k] = v;
 
-        // Request an authorization code from the OAuth server. Subsequently,
-        // the authorization code may be exchanged for an access token.
-        reqParams["response_type"] = "code";
+        reqParams["response_type"] =
+            (provider.options & OAuthProvider.Option.tokenResponseType) ? "token" : "code";
```

Comment:
This looks like you're trying to use the OAuth2 implicit flow, which was excluded from this library because it is meant to be used by in-browser apps only, and D is not a language used to created such apps.

Please see the instructions for VK on https://vk.com/dev/authcode_flow_user

Reply:
No, I use it for auth of parsing tool. Tool generates URL, user clicks to it and then copy and pastes token response URL from browser into this tool.